scripts/setup_truefoundry_cluster.sh

```diff
@@ -208,35 +154,24 @@ main() {
     make_request "GET" "${CONTROL_PLANE_URL}/api/svc/" "" "200" >/dev/null || \
         handle_error "Platform health check failed"
 
-    # Get environment and create cluster
-    environment_name=$(get_environment_name)
-    log_info "main: Found environment: ${environment_name}"
-
     # Setup provider account if configured and cluster type is not generic
     if [ "${CLUSTER_TYPE}" != "generic" ]; then
         setup_provider_account
     fi
-    cluster_manifest=$(generate_cluster_manifest "${environment_name}")
-    log_info "main: ${cluster_manifest}"
-    cluster_id=$(create_cluster "${cluster_manifest}")
-    cluster_token=$(get_cluster_token "${cluster_id}")
 
-    # Get tenant information
-    tenant_name=$(get_tenant_name)
+    cluster_id=$(create_cluster)
+    cluster_token=$(get_cluster_token "${cluster_id}")
 
     log_info "main: cluster_id=$cluster_id"
     log_info "main: cluster_token=$cluster_token"
-    log_info "main: tenant_name=$tenant_name"
 
     # Output only the final JSON result to stdout
     jq -n \
         --arg cluster_id "$cluster_id" \
         --arg cluster_token "$cluster_token" \
-        --arg tenant_name "$tenant_name" \
         '{
             cluster_id: $cluster_id,
             cluster_token: $cluster_token,
-            tenant_name: $tenant_name
```

Comment:
why are we removing this?

Reply:
This is now taken from `get_environment.sh` file. This script takes the environment details, tenant name and it helps in generating the provider config